src/components/what-we-do/platforms/platforms.js

```diff
@@ -40,7 +41,12 @@ const Platforms = ({ title, platforms }) => {
               <PlatformTitle>{platform.platformTitle}</PlatformTitle>
               <P>{platform.blurb}</P>
               <a href={platform.ctaLink} target="_blank" rel="noreferrer">
-                <PlatformButton primary={true} aria-label={platform.ctaTitle}>
+                <PlatformButton
+                  padding={"17px 55px"}
+                  boxshadow={`1.5px 1.5px 2px ${shadowGray}`}
```

Comment:
In general I recommend using variants instead of passing around numbers, and let the styled component handle figuring out which pixel values to use. This keeps magic numbers out of your code and sets you up for better consistency across components.